### ISSUE TITLE
Filter tab data is weird.

### ISSUE THREAD

**Issue by MercerK** [POE-Addon-Launcher/PoE-Addon-Launcher#4]
![image](https://user-images.githubusercontent.com/13123338/45985395-0a64a300-c034-11e8-87bc-f3998454d1b8.png)

Only have Neversink installed, but according to this, all are.

**Comment by POE-Addon-Launcher** [comment]
Yeah, that's a known issue, that I'm working on thanks for still reporting it though.
